docs/articles/basics.md

```diff
@@ -76,13 +76,17 @@ then it is trivial for someone to gain root access to the host where the
 daemon is running.
 
 Similarly, the Docker client can use `-H` to connect to a custom port.
+The Docker client will default to connecting to `unix:///var/run/docker.sock`
+on Linux, and `tcp://127.0.0.1:2376` on Windows.
 
 `-H` accepts host and port assignment in the following format:
 
-    tcp://[host][:port][path] or unix://path
+    tcp://[host:][port][path] or unix://path
```

Comment:
Also the comment `-H accepts host and port assignment in the following format:` is incorrect in the light of the use of path, regardless of its correctness or not in the tcp case. It's definitely not a port assignment in the unix:/// case.